src/components/Todo/index.js

```diff
@@ -41,8 +41,12 @@ export default class Todo extends React.Component {
           <DropdownItem onClick={() => onDelete(todo.id)}>
             <span className="has-text-danger">Delete</span>
           </DropdownItem>
+          <DropdownDivider/>
+            <DropdownItem onClick={() => onSnooze(todo, 1, console.log(todo.newTodoDate))}>Snooze 1 day</DropdownItem>
```

Comment:
You have an indentation issue. `DropdownDivider` is an item inside of `Dropdown`, it should not provoke and indentation after it.

So 
```jsx
<Dropdown>
  <DropdownItem onClick={() => onEdit(todo)}>Edit</DropdownItem>
  <DropdownItem onClick={() => onDelete(todo.id)}>
    <span className="has-text-danger">Delete</span>
  </DropdownItem>
  <DropdownDivider/>
  <DropdownItem onClick={() => onSnooze(todo, 1, console.log(todo.newTodoDate))}>Snooze 1 day</DropdownItem>
  <DropdownItem onClick={() => onSnooze(todo, 3)}>Snooze 3 days</DropdownItem>
  <DropdownItem onClick={() => onSnooze(todo, 7)}>Snooze 1 week</DropdownItem>
  <DropdownItem onClick={() => onSnooze(todo, 21)}>Snooze 3 weeks</DropdownItem>
</Dropdown>
```